.taskcluster.yml

```diff
@@ -76,19 +81,28 @@ tasks:
           deadline: {$fromNow: '2 hours'}
           expires: ${expires_in}
           provisionerId: aws-provisioner-v1
-          workerType: ${decision_worker_type}
+          workerType: mobile-${repo_trust_level}-decision
           priority: ${tasks_priority}
           requires: all-completed   # Must be explicit because of Chain of Trust
           retries: 5
           scopes:
-            - queue:create-task:${tasks_priority}:aws-provisioner-v1/${build_worker_type}
+            - queue:create-task:${tasks_priority}:aws-provisioner-v1/mobile-${repo_trust_level}-b-andrcmp
+            - queue:create-task:${tasks_priority}:aws-provisioner-v1/mobile-${repo_trust_level}-images
+            - queue:route:index.project.mobile.android-components.cache.level-${repo_trust_level}.docker-images.v1.*
```

Comment:
Not blocking here, but I wonder if it would be possible to just have this be the appropriate `assume:repo:github.com/mozilla-mobile/android-components:*` role.